x-pack/plugins/lens/public/xy_visualization/xy_suggestions.ts

```diff
@@ -241,9 +238,13 @@ function getSuggestionsForLayer({
     return visualizationTypes
       .map((visType) => {
         return {
-          ...buildSuggestion({ ...options, seriesType: visType.id as SeriesType }),
+          ...buildSuggestion({
+            ...options,
+            seriesType: visType.id as SeriesType,
+            // explicitly hide everything besides stacked bars, use default hiding logic for stacked bars
+            hide: visType.id === 'bar_stacked' ? undefined : true,
```

Comment:
What's the particular difference here with `visType.id !== 'bar_stacked'`?

Reply:
Stacked bars is the visible xy suggestion (e.g. when on pie):
<img width="674" alt="Screenshot 2020-11-20 at 15 20 19" src="https://user-images.githubusercontent.com/1508364/99810391-f2b9d400-2b43-11eb-95bc-06c733504fcc.png">

it's the "default" xy sub type, that's why we are showing it